mne/channels/_standard_montage_utils.py

```diff
@@ -50,13 +50,16 @@ def _easycap(basename, head_size):
 
     ch_pos = montage._get_ch_pos()
 
-    nasion = np.concatenate([[0], ch_pos['Fpz'][1:]])
-    lpa = np.mean([ch_pos['FT9'],
-                   ch_pos['TP9']], axis=0)
-    lpa *= head_size / np.linalg.norm(lpa)  # on sphere
-    rpa = np.mean([ch_pos['FT10'],
-                   ch_pos['TP10']], axis=0)
-    rpa *= head_size / np.linalg.norm(rpa)
+    if basename == 'easycap-M1.txt':
+        nasion = np.concatenate([[0], ch_pos['Fpz'][1:]])
+        lpa = np.mean([ch_pos['FT9'], ch_pos['TP9']], axis=0)
+        lpa *= head_size / np.linalg.norm(lpa)  # on sphere
+        rpa = np.mean([ch_pos['FT10'], ch_pos['TP10']], axis=0)
+        rpa *= head_size / np.linalg.norm(rpa)
+    elif basename == 'easycap-M10.txt':
+        nasion = lpa = rpa = None
```

Comment:
This change does not look like reading one versus the other but rather whether or not the nasion/LPA/RPA are defined for the montage. Can you explain a bit?

Reply:
Selecting the right cap happens on line 130/133 in the next hunk below. This here is a secondary change -- the M10 montage has different channel names so inferring the nasion can't be done with the same code. Based in [this](https://www.easycap.de/wp-content/uploads/2018/02/Easycap-Equidistant-Layouts.pdf) mapping (p. 5) the same heuristic could be applied though.

Reply:
Without inferring fiducials:
![map-pre](https://user-images.githubusercontent.com/145771/72608221-3d4d3d00-38f0-11ea-8531-f43095abc612.png)
With inferring fiducials:
![map-post](https://user-images.githubusercontent.com/145771/72608243-46d6a500-38f0-11ea-9099-966088dac7af.png)

Reply:
Is it worth shifting the inferred LPA/RPA to fit the head more precisely?

Reply:
The best thing to do would be to figure out where they are supposed to be in the montage, rather than judge based on the plot

Reply:
> The best thing to do would be to figure out where they are supposed to be in the montage, rather than judge based on the plot

I agree, do you know how to do that? I don't have a physical cap... And that would also apply equally to the M1 montage.

Reply:
At least for M1 I think that based on a standard 1020 layout definition we should be able to figure it out. It looks like like T3 and T4 being 10% of the way between LPA/RPA and Cz, and Fpz / Oz are 10% up from Nasion/Inion:

http://chgd.umich.edu/wp-content/uploads/2014/06/10-20_system_positioning.pdf

I'd first verify that the spacings in that doc are (at least approximately) accurate for M1 and then go from there.

For M10 we might want to see if the numerical names have readable equivalents (e.g., "10" is "Cz" or whatever) and go from there

Reply:
The first plot looks much better - that seems to be the exact problem I have in plots in #7193.
@larsoner, @christianbrodbeck - I have digitized positions for this cap, I think, I can take a look.

Reply:
(ok, the problem is different in the PR, there it is yet another eeg montage. But I should have easycap digitized too)

Reply:
That would be helpful if you can check. And we should check that the percentage arc length math works, too. If it doesn't we should figure out how much they differ and choose which one seems correct.

Reply:
Thanks @larsoner! These are the new montages after adding fiducials based on the spherical coordinate system:

![map-rad](https://user-images.githubusercontent.com/145771/72666578-d019d480-39e1-11ea-9140-24ed81c08825.png)
![m10-rad](https://user-images.githubusercontent.com/145771/72666582-d3ad5b80-39e1-11ea-9da2-8195dbba61f6.png)